common/src/ethereum.rs

```diff
@@ -223,6 +243,14 @@ impl Wallet {
                 .context("Could not connect to Ledger device")?;
 
             Ok(Wallet::Ledger(signer))
+        } else if options.walletconnect {
+            let signer = WalletConnect::new()
+                .map_err(|_| anyhow!(""))
+                .context("Failed to create WalletConnect client")?
```

Comment:
Perhaps move this error string into the `anyhow!` right above, no need for `context()` then.

Reply:
https://github.com/radicle-dev/radicle-cli/commit/ce9ad2d97fec16a5ef720cbb82ae87a96f6f5e28#diff-799f6adcb3a8a0469e7950f80331d4ffcb715140d420d1f4b25bad88ee61adc9R248